xml/System.IO/File.xml

```diff
@@ -6474,7 +6474,7 @@ It is not possible to change the compression status of a <xref:System.IO.File> o
       <Docs>
         <param name="path">The file to write to.</param>
         <param name="bytes">The bytes to write to the file.</param>
-        <summary>Creates a new file, writes the specified byte array to the file, and then closes the file. If the target file already exists, it is overwritten.</summary>
+        <summary>Creates a new file, writes the specified byte array to the file, and then closes the file. If the target file already exists, it is truncated and overwritten.</summary>
```

Comment:
Wdyt? This way we can have the description in natural English and in a technical wording that evokes curiosity.

```suggestion
        <summary>Creates a new file, writes the specified byte array to the file, and then closes the file. If the target file already exists, overwrites any existing content (truncates to length zero).</summary>
```

Reply:
Definitely should be "first truncating to length 0" rather than "truncates to length 0", but otherwise I'm not opposed to that.